docs/content/grafana/add_grafana_dashboard.md

```diff
@@ -41,4 +41,11 @@ $ cd grafana/overlays/moc/smaug/dashboards
 $ kustomize edit add resource ${DASHBOARD_FOLDER}
 ```
 
+### Backing Up Your Dashboard:
+Grafana redeploy frequently, whether it be a operator update or a new dashboard has been added via Pull Request. Please make sure to backup your dashboards.
+
+To export your dashboard click the `share icon button` at the top dashboard menu. Click the `Export` tab and now you can now save a JSON file of your dashboard.
```

Comment:
I think you're misusing inline code formatting here.

The phrase `share icon button` doesn't need to be formatted as inline code (or quoted at all, really). Ideally, we would actually include the share icon inline here, because people may not be sure what they're looking for.

The word Export should just use regular quotes (`"Export"`).